app.rb

```diff
@@ -0,0 +1,24 @@
+require 'sinatra/base'
+require './chitter.rb'
+
+class ChitterManager < Sinatra::Base
+
+  get '/' do
+    'Welcome to Chitter'
+  end
+
+  get '/allpeeps' do
+    @chitter = Chitter.all
+    erb :allpeeps
+  end
+
+  post '/allpeeps' do
+    Chitter.peep(@message = params[:message])
+    redirect '/allpeeps'
```

Comment:
Excellent use of a post and redirect. 
It's also great that you are delegating to your model object to handle the connection to the database